bin/rector.php

```diff
@@ -48,6 +49,16 @@
     $rectorContainerFactory = new RectorContainerFactory();
     $container = $rectorContainerFactory->createFromBootstrapConfigs($bootstrapConfigs);
 } catch (SetNotFoundException $setNotFoundException) {
+    // possible custom set list
+    $match = Strings::match($setNotFoundException->getMessage(), '#Set (".*") was not found#');
+    if ($match) {
+        $path = realpath(trim($match[1], '"'));
+        if ($path) {
+            // eg : /Users/samsonasik/www/laminas-mvc-skeleton/vendor/laminas/laminas-servicemanager-migration/config/rector/set/laminas-servicemanager-40.php
+            echo $path;die;
+        }
+    }
```

Comment:
Hi @TomasVotruba , this is where the path is catched when we define custom `SetList` class, do you have idea to make it work to read `$path` as load config from it ?

ref https://github.com/rectorphp/rector/issues/5891
ref https://github.com/laminas/laminas-servicemanager-migration/pull/2#issuecomment-801530436

Reply:
@TomasVotruba it works 🎉 , I updated to try re-create container with path found https://github.com/rectorphp/rector/pull/5894/commits/d96afeca8d4d8570ee341eddef88288649b6194b